File: app/src/main/java/com/seankim/zomatosearchex/Model/RspRestaurantModel.java
package com.seankim.zomatosearchex.Model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RspRestaurantModel {
    @SerializedName("restaurant")
    @Expose
    RspRestaurantBeanModel RspRestaurantBeanModel;

    public RspRestaurantBeanModel getRspRestaurantBeanModel() {
        return RspRestaurantBeanModel;
    }

    public void setRspRestaurantBeanModel(RspRestaurantBeanModel rspRestaurantBeanModel) {
        RspRestaurantBeanModel = rspRestaurantBeanModel;
    }
}

File: app/src/main/java/com/seankim/zomatosearchex/Model/RspRestaurantModel.kt
package com.seankim.zomatosearchex.Model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class RspRestaurantModel {
    @SerializedName("restaurant")
    @Expose
    lateinit var rspRestaurantBeanModel: RspRestaurantBeanModel
}
